go.mod

```diff
@@ -36,4 +33,5 @@ require (
 	golang.org/x/net v0.0.0-20190912160710-24e19bdeb0f2
 	golang.org/x/text v0.3.2 // indirect
 	gopkg.in/natefinch/npipe.v2 v2.0.0-20160621034901-c1b8fa8bdcce // indirect
+	gopkg.in/urfave/cli.v1 v1.20.0 // indirect
```

Comment:
Are the mod changes needed for linting?

Reply:
I did a go mod tidy before committing the changes, that cleans up the go.mod and go.sum files.
in previous commits, this would have not been done. Hence the files are changing.

If we want to retain the same versions of go.mod and go.sum, I could also revert this change.

Reply:
Ok fine